lib/include/elements/app.hpp

```diff
@@ -9,7 +9,7 @@
 #include <string>
 #include <infra/support.hpp>
 
-#if defined(__linux__)
+#if defined(_HOST_UI_LIBRARY_GTK_)
```

Comment:
Oh, sorry, a single underscore + caps is also illegal: https://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier

"All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use."

Reply:
I suggest something like ELEMENTS_HOST_UI_LIBRARY_GTK

Reply:
Sorry about that.